packages/catalog-model/examples/domains/playback-domain.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: backstage.io/v1alpha1
+kind: Domain
+metadata:
+  name: playback
+  description: Everything related to audio playback
+spec:
+  owner: User/frank.tiernan
```

Comment:
By intention a user, to test that case too

Reply:
shouldnt't this be `user:frank.tiernan`? I thought the slash was for namespaces ...